<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>645</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>431</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(35, 35, 35);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>431</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 86, 126);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="nome">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(222, 222, 222);</string>
       </property>
       <property name="placeholderText">
        <string>Nome</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="senha">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>210</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(222, 222, 222);</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string extracomment="Digite sua senha">Digite sua senha</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="email">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>160</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(222, 222, 222);</string>
       </property>
       <property name="placeholderText">
        <string>Digite seu e-mail</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <strikeout>true</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Verdana&quot;;
text-decoration: line-through;
text-decoration: line-through;</string>
       </property>
       <property name="text">
        <string>empresa aqui</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cadastrarse">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>330</y>
         <width>151</width>
         <height>23</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS UI Gothic&quot;;
background-color: rgb(0, 122, 179);
alternate-background-color: rgb(85, 170, 255);</string>
       </property>
       <property name="text">
        <string>Cadastrar-se</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="confirmasenha">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>260</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(222, 222, 222);</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Digite novamente a senha</string>
       </property>
      </widget>
      <widget class="QLabel" name="warning_confirmasenha">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>280</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(218, 9, 12);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="warning_senha">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>230</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(218, 9, 12);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="warning_email">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>180</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(218, 9, 12);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="warning_nome">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(218, 9, 12);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="voltar">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>370</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS UI Gothic&quot;;
background-color: rgb(0, 122, 179);
alternate-background-color: rgb(85, 170, 255);</string>
       </property>
       <property name="text">
        <string>Voltar</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
